official/utils/misc/model_helpers.py

```diff
@@ -53,3 +54,25 @@ def past_stop_threshold(stop_threshold, eval_metric):
     return True
 
   return False
+
+
+def set_random_seed(seed):
+  """Sets the random seeds for available RNGs.
+  This seeds RNGs for python's random and for Tensorflow. The intended
+  use case is for this to be called exactly once at the start of execution
```

Comment:
Which "execution" does it mean here? The execution of the training process? Or the execution of one Session? Or others? Does it act the same in the distributed environment (Multiple GPUs) vs. one single GPU?